app/views/content/form/_rich_text_input.html.erb

```diff
@@ -2,7 +2,7 @@
   content_name = f.object.class.content_name
 
   field_id = "#{content_name}_#{field[:id]}"
-  value = if f.object.respond_to?(field[:id].to_sym)
+  value ||= if f.object.respond_to?(field[:id].to_sym)
```

Comment:
:eyes:

Reply:
This is to allow overriding `value` in the case of e.g. creating a page from a document entity, where the `name` field is prefilled on `#new`.